samples/excel/89-preview-apis/document-properties.yaml

```diff
@@ -0,0 +1,118 @@
+order: 6
+id: excel-document-properties
+name: Document properties
+description: Get and set document properties.
+host: EXCEL
+api_set:
+    ExcelApi: 1.7
+script:
+    content: |+
+        $("#get-doc-properties").click(() => tryCatch(getDocProperties));
+        $("#set-get-doc-properties").click(() => tryCatch(setGetDocProperties));
+
+        async function getDocProperties() {
+            await Excel.run(async (context) => {
+                let docProperties = context.workbook.properties;
+
+                // Load the read-only document properties.
+                docProperties.load("author, lastauthor, revisionnumber, creationdate");
+
+                await context.sync();
+
+                console.log("Author  :" + docProperties.author);
+                console.log("Last author  : " + docProperties.lastAuthor);
+                console.log("Revision number : " + docProperties.revisionNumber);
+              
+                OfficeHelpers.UI.notify("Workbook creation date : " + docProperties.creationDate.toDateString());            
+            });
+        }
+
+        async function setGetDocProperties() {
+            await Excel.run(async (context) => {
+                let titleValue = "Excel document properties API";
```

Comment:
We generally use `const` instead if `let` in our snippets for variables that won't get written to again. But that's somewhat controversial as guidance, so you can leave it if you prefer `let`.